.azuredevops/platformPipelines/platform.updateReadMe.yml

```diff
@@ -23,9 +23,12 @@ variables:
     value: 'utilities/tools/platform/Set-ReadMeModuleTable.ps1'
   - name: relativeScriptPath_SetModuleOverviewTable
     value: 'utilities/tools/platform/Set-ModuleOverviewTable.ps1'
+  - name: relativeScriptPath_SetPlatformPipelinesTable
+    value: 'utilities/tools/platform/Set-PlatformPipelinesTable.ps1'
 
 jobs:
-  - job: Update_module_tables
+  - job: Update_status_tables
+    displayName: Update status tables
```

Comment:
Same comment of the GH workflow counterpart. Not related to this line but to task on line 41. For `displayName` I'd suggest `'Update Module Pipelines table'`

Reply:
Updated in both